docs/DeveloperGuide.md

```diff
@@ -338,18 +338,18 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* freelance pre-university home tutors
-* has a need to manage a significant number of students, their contacts and relevant information
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* Freelance pre-university home tutors
+* Has a need to manage a significant number of students, their contacts and relevant information
+* Prefer desktop apps over other types
+* Can type fast
```

Comment:
I think this might not be required, since we mentioned "Prefers typing rather than mouse interactions"